### DISCUSSION TITLE
Is it possible to add a custom intercom-like widget using the new Gitbook Integration Platform?

### DISCUSSION THREAD

**Discussion by MantisClone** [GitbookIO/community#315] [0. General bugs and questions] [answered]
I want to add [WidgetBot](https://widgetbot.io/) to my [Documentation hosted by Gitbook](https://docs.request.network/). Is this possible using the [Gitbook Integration Platform](https://developer.gitbook.com/).

Related Discussions:
* https://github.com/orgs/GitbookIO/discussions/43 - Crisp Integration, another Intercom-like widget.

**Comment by addisonschultz** [comment, marked as answer]
Hi @MantisClone - This is unfortunately not possible at the moment due to security concerns with developing integrations like this. We know it's a highly requested feature request - so have it on our radar to explore in the future.
